### ISSUE TITLE
provide build time context to runtime 

### ISSUE THREAD

**Issue by gauravphoenix** [dassana-io/dassana#264]
One of the most painful tasks in the cloud world too is to chase down people who own resources. If a team is using IAC- be it TF or CFT, and using some CICD tooling such as GHA, we can provide a GHA action that can be run after the CICD step has just executed the TF/deployed TF. This action will do a HTTP POST to the Dassana Rest API with the details of github repo, commit details etc. The REST API will store this information in the s3 bucket with key e.g. 
`s3://dassana-bucket/ec2/i-foo` and the value being the HTTP POST details. 

Since Dassana automatically does alert normalization, it will be trivial to pull details from this s3 bucket and provide information on who made the last change etc. 

Even if an alert is not being investigated, since Dassana workflows can be run directly, we can provide a workflow which will take input as resource id, service etc and give out details. More details [here](https://docs.dassana.io/docs/getting-started/hello-world/#resource-context). 

In short, given a cloud resource (service, resource type and resource id), we will be able to provide a capability whereby one can do a HTTP POST like this- 

```bash
curl --location --request POST 'https://<rest-api-id>.execute-api.us-east-1.amazonaws.com/v1/run?workflowId=get-detaiks\

--header 'x-api-key: <YOUR-API-KEY-GOES-HERE>' \
--data-raw '{
  "service":"ec2",
  "resourceType:"instance",
  "resourceId":"i-123"
}'

```

and get build time details.
